client/client_test.go

```diff
@@ -61,6 +65,38 @@ func (c *testSetupSecureClient) GetRootCertificateAuthorities() ([]*x509.Certifi
 }
 
 func NewTestSecureClient() (*client.Client, error) {
+	return newTestSecureClient(test.IdentityIntermediateCA, test.IdentityIntermediateCAKey)
+}
+
+func NewTestSecureClient1() (*client.Client, error) {
```

Comment:
I'm thinking about a more descriptive name than `NewTestSecureClient1`.  Maybe something like `NewTestSecureClientWithGeneratedCertificate`, though that's rather long.

Reply:
Renamed